README.md

```diff
@@ -1,58 +1,55 @@
+- pip, virtualenv, homebrew (OSX only)
+- Postgres >= 9.4
+  - OSX recommended install method: [Postgres.app](http://postgresapp.com/)
+- Python 3
+  - OSX recommended install method: Homebrew `brew install python3`
+- Docker (_optional, but strongly recommended_)
+  - OSX recommended install method: [Download from Docker website](https://docs.docker.com/mac/)
+- docker-compose (_if you're using Docker, this is required_)
+  - Recommended install: pip `pip install docker-compose`
 
-    pip install docker-compose 
-    docker-compose up
+## Getting Started
 
-### (OSX only) Create your docker machine:
+1. Create and activate a virtualenv in the root folder of the Micromasters project directory.
+1. Install the python dependencies ``pip install -r requirements.txt``
 
-    docker-machine create default
-    docker-machine start default
-    docker-machine env default
-    eval "$(docker-machine env default)"
+#### (OSX only) Create your docker container:
 
-These commands create a Docker container named ``default``, start the
+These commands create a Docker container named ``mm``, start the
 container, and configure environment variables to facilitate communication
-with the ``edX`` instance.
+with the edX instance.
 
+    docker-machine create mm
+    docker-machine start mm
+    # 'docker-machine env (container)' prints export commands for important environment variables
+    eval "$(docker-machine env mm)"
 
-This will set up a near production-ready containerized development 
-environment that runs migrations, with the Django development server 
-running on port ``8079``.
-
-To run one-off commands, like shell, you can run:
-
-    docker-compose run web python manage.py shell
+_**NOTE:** Setting an environment variable called
+**MM_CONTAINER_NAME** will allow shell/python scripts and one-off commands
+to easily work with your Docker container. Add this to your shell config
+file (.bashrc, et al) with your desired container name:
+``export MM_CONTAINER_NAME='mm'``_
 
-or to create root user:
-
-    docker-compose run web python manage.py createsuperuser
-
-
-## For OS X Development
+#### (OSX only) Enable file syncing between the host machine and Docker:
 
 Due to issues using Docker in OSX, development is more efficient if you
 install [docker-osx-dev](https://github.com/brikis98/docker-osx-dev).
-docker-osx-dev synchronizes host file-system changes with the Docker
-container. If you have [``homebrew``](http://brew.sh/) installed, you can 
-install ``docker-osx-dev`` by typing ``make`` in the root of the Micromasters
-project directory. The make file will install or update Docker using Homebrew
-and go on to install ``docker-osx-dev``.
+This synchronizes host file-system changes with the Docker container.
+
+First, set up ``docker-osx-dev`` via shell script. You can specify a container
+name as an argument or in the **MM_CONTAINER_NAME** env. variable. If the script
+gets neither, the container name is assumed to be 'default'.
```

Comment:
i actually got rid of that handling of container name entirely. forgot to update this accordingly. good catch :)